hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -571,15 +534,13 @@ def check_ungrouped_files(reason=''):
 
     ungrouped_files = _execute_shell_command("df --local -P | awk 'NR!=1 {print $6}' | xargs -I '{}' find '{}' -xdev -nogroup 2>/dev/null").strip()
```

Comment:
Even on local filesystems, has impact of these commands been tested, say on a 2.5TB fileserver?

Reply:
Will try to do this. Opened a new ticket for the same. But as of now we are not using this function in any profile.

Reply:
My aim was that doing blanket _find_ commands on whatever comes out of _df_ (or other commands) is going to end up running over data we didn't intend to. It would be preferable to be more limited on the set of paths that find traverses.

Reply:
Got it. Make sense.